### PR TITLE
Removing styles

### PR DESCRIPTION
**Pull request by kayode94**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
Removes orange line from home page and changes color scheme of the side bar

## Type of change
UI/UX

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by dancingheart714 — APPROVED** [review]
These changes appear to be good. Code is fine. I’ll be curious to see how the colors render on the screen!

**Review by Gavin-Rilee — APPROVED** [review]
Love the changes :)

File: src/components/pages/DataHub/DataHub.js [hunk 1]
```diff
@@ -207,9 +207,7 @@ const DataHub = props => {
         added, more data can be visualized.
       </h3>
       <h2 className="h1Styles">Approvals vs Denials Nationwide</h2>
-      <p className="divider">
-        <Icon component={() => <img src={OrangeLine} alt="divider icon" />} />
-      </p>
+      <p className="divider"></p>
```
Comment (CrashOverride1987) [inline src/components/pages/DataHub/DataHub.js:210]: What is the purpose of this p tag?

**Review by DanielBrannon — DISMISSED** [review]
The p tag in src/components/pages/DataHub/DataHub.js that was surrounding the orange line Icon, no longer serves a purpose. please remove it

File: src/components/pages/DataHub/DataHub.js (hunk 1 above)
Comment (kayode94) [reply src/components/pages/DataHub/DataHub.js:210]: Not sure, I didn't add that and I didn't notice it before. I will remove it on the next push

Comment (CrashOverride1987) [reply src/components/pages/DataHub/DataHub.js:210]: If you look, the p tag was there before, you just removed the Orange Icon component and left it's parent component, that no longer serves a purpose

**Review by Humza23 — APPROVED** [review]
Good work Kayode. Good catch Daniel for the extra <p> tag.

**Pull request merged**